normalizer/docker/parser.py

```diff
@@ -41,6 +43,42 @@ def __init__(
 
     def __str__(self):
         return self.content
+    
+    def get_build_args_envs(self, build_args_envs): 
+        build_args = '';
+        for index, item in enumerate(build_args_envs):
+            build_args += dedent(
+            f"""\
+            ARG {item}
+            """
+        )
+        build_envs = '';
+        for index, item in enumerate(build_args_envs):
+            build_envs += dedent(
+            f"""\
+            ENV {item}=${item}
+            """
+        )
+        return (build_args, build_envs)
```

Comment:
Is `ENV` necessary? `--build-arg` is enough for this case, am I missing something?

Reply:
When we use pip to install some dependencies in requirements.txt, if a variable is set in requirements.txt, pip will query this variable from the environment variable, so ENV must be set, --build-arg just passes in the parameter